### PR TITLE
Update how-to-write-guest-blog-posts-for-mattermost-apps-and-services.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Trailing spaces.

### CODE REVIEW

File: operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-write-guest-blog-posts-for-mattermost-apps-and-services.md [hunk 1]
```diff
@@ -17,15 +17,14 @@ Please provide the following for Mattermost by email:
...
+A link to the final draft of the blog post will be sent to you by email prior to publication, so that you can offer feedback and recommendations on it.
 
 ### Mattermost Guest Blog Post Coordinator
 
 Follow these guidelines to develop a clear and concise guest blog post:
 
-* Begin the article with a short bio and introduction of the author\(s\). Include a link to the guest company’s website the first time it is mentioned in the text.
+* Begin the article with a short bio and introduction of the author\(s\). Include a link to the guest company’s website the first time it's mentioned in the text.
 * Include a screenshot of the guest company’s logo in the post.
 * At the end of the post, include a short summary of the guest company as well as a call for readers to share feedback.
```
Comment (justinegeffen) [inline operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-write-guest-blog-posts-for-mattermost-apps-and-services.md:28]: @aedott; would this be a link to the forum or an email address, or a link to a channel? Or all three?

File: operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-write-guest-blog-posts-for-mattermost-apps-and-services.md [hunk 2]
```diff
@@ -17,15 +17,14 @@ Please provide the following for Mattermost by email:
 
 If you provide screenshots, please help ensure they meet our [screenshot development guidelines](https://handbook.mattermost.com/operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-create-screenshots-and-gifs).
 
-A link to the final draft of the blog post will be sent to you by email, so that you can offer feedback and recommendations on it.
+A link to the final draft of the blog post will be sent to you by email prior to publication, so that you can offer feedback and recommendations on it.
```
Comment (justinegeffen) [inline operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-write-guest-blog-posts-for-mattermost-apps-and-services.md:20]: @aedott, please confirm that this is done prior to publication. Thanks!

**Review by amyblais — APPROVED** [review]

**Comment by justinegeffen** [comment]
@aedott, following up on this PR for review. :)

**Comment by justinegeffen** [comment]
Gentle reminder on this PR, thanks @aedott! :)

File: operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-write-guest-blog-posts-for-mattermost-apps-and-services.md (hunk 2 above)
Comment (aedott) [reply operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-write-guest-blog-posts-for-mattermost-apps-and-services.md:20]: Yes, all contributors have a chance to review the final draft before it goes live!

File: operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-write-guest-blog-posts-for-mattermost-apps-and-services.md (hunk 1 above)
Comment (aedott) [reply operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-write-guest-blog-posts-for-mattermost-apps-and-services.md:28]: It should be a link to the homepage of the guest company's site, e.g. www.mattermost.com. There might be some exceptions, but those would need to be addressed on a case-by-case basis.

**Comment by justinegeffen** [comment]
@amyblais, you can ignore the review request. :)

**Pull request merged**